akka-http/src/main/scala/akka/http/javadsl/server/Directives.scala

```diff
@@ -26,4 +28,96 @@ object Directives extends AllDirectives {
 
   @varargs override def getFromBrowseableDirectories(directories: String*): Route =
     super.getFromBrowseableDirectories(directories: _*)
+
+  /**
+   * Partially applying functions for the Java World!
```

Comment:
Sure, I will give my best explaining what it means and the in-line example is a good idea :slightly_smiling_face: